Modules/xSharePoint/DSCResources/MSFT_xSPFarmSolution/MSFT_xSPFarmSolution.schema.mof

```diff
@@ -0,0 +1,27 @@
+/*
+**Description**
+
+This resource will deploy a farm solution to a farm. 
+A version will be stored in the property bag to determin later if the correct version is installed.
+
+**Example**
+
+    xSPFarmSolution SampleWsp
+    {
+        Name                 = "MySolution.wsp"
+        LiteralPath          = "C:\src\MySolution.wsp"
+        Version              = "1.0.0"
+        PsDscRunAsCredential = $InstallAccount
+    }
+*/
+[ClassVersion("1.0.0.0"), FriendlyName("xSPFarmSolution")]
+class MSFT_xSPFarmSolution : OMI_BaseResource
+{
+    [Key] string Name;
+    [Required] string LiteralPath;
+    [Write] string WebApplications[];
```

Comment:
Yes. See line 218 to 236. You can specify one or more web application. If you do not specify any it defaults to AllWebApplications if it has "ContainsWebApplicationResource". Global Solutions are installed without the WebApplication Parameter.